snowpack/src/scan-imports.ts

```diff
@@ -155,23 +152,38 @@ function parseFileForInstallTargets({
   locOnDisk,
   baseExt,
   contents,
-}: SnowpackSourceFile): InstallTarget[] {
+}: SnowpackSourceFile<string>): InstallTarget[] {
+  const relativeLoc = path.relative(process.cwd(), locOnDisk);
+
   try {
     switch (baseExt) {
       case '.css':
       case '.less':
       case '.sass':
       case '.scss': {
+        logger.debug(`Scanning ${relativeLoc} for imports as CSS`);
         return parseCssForInstallTargets(contents);
       }
       case '.html':
       case '.svelte':
       case '.vue': {
+        logger.debug(`Scanning ${relativeLoc} for imports as HTML`);
         return parseJsForInstallTargets(extractJSFromHTML({contents, baseExt}));
       }
-      default: {
```

Comment:
This was the source of the bug: we were trying to parse binary files for imports, but hanging on the error.